#this message enables the mosfet power interface for the PCA embedded in the ONOHAT

bool value
---
bool result